assets/javascript/cards.js

```diff
@@ -582,91 +487,75 @@ function initializeLarge(i, term) {
-    };
+//wiki blurb==================================================================================================================================================
+var blurb;
 
-    //$('.scrollspy').scrollSpy();
 
-    mapLarge = new google.maps.Map(document.getElementById('mapLarge'),
-        mapOptionsLarge);
-}
+function getBlurb() {
+    searchTerms.forEach(function (term, i) {
+        //gets page id
+        var url = "https://en.wikipedia.org/w/api.php?origin=*&action=query&list=search&srsearch=" + term + "&srlimit=1&format=json";
 
 
-// $("#multipleCards").on("click", ".nonactiveCard", function () {
+        //gets blurb off of page
+        var blurbUrl = "https://en.wikipedia.org/w/api.php?origin=*&format=json&action=query&prop=extracts&exintro&explaintext&exsentences=6&redirects=1&titles=" + term;
 
-//     $(".activeCard").removeClass("m12");
-//     $(".activeCard").addClass("m6");
-//     //console.log($(".activeCard").attr("cardNum"));
-//     $(".image" + $(".activeCard").attr("cardNum")).addClass("m6");
-//     $(".image" + $(".activeCard").attr("cardNum")).removeClass("m3");
-//     $(".activeCard").removeClass("activeCard");
-//     $("#displayCard").empty();
+        //gets unique page id
+        $.ajax({
+            url: url,
+            method: "GET"
+        })
+            .then(function (response) {
+                var pageID = response.query.search[0].pageid;
 
-//     $(this).clone().addClass("activeCard").removeClass("nonactiveCard").appendTo("#displayCard");
-//     $(".activeCard").removeClass("m6");
-//     $(".activeCard").addClass("m12");
-//     //console.log($(".activeCard").attr("cardNum"));
-//      $(".image" + $(".activeCard").attr("cardNum")).removeClass("m6");
-//      $(".image" + $(".activeCard").attr("cardNum")).addClass("m3");
+                //uses unique page id to get blub
+                $.ajax({
+                    url: blurbUrl,
+                    method: "GET"
+                })
+                    .then(function (response) {
 
+                        console.log(response);
+                        blurb = response.query.pages[pageID].extract;
 
-// })
+                        blurb = blurb.replace('(listen)', '');
+                        blurb = blurb.replace('(listen);', '');
+                        blurb = blurb.replace('(listen),', '');
+                        blurb = blurb.replace('( )', '');
+                        blurb = blurb.replace('()', '');
 
 
+                        $(`#blurbHere${i}`).append(blurb);
 
+                    })
 
+            })
+    })
+}
 
 
 
 
-// function assignInitialDisplayCard(i) {
-//     if (i <= 0) {
-//         console.log(`image${i}`)
-//         $(`.image${i}`).removeClass("m6")
-//         $(`.image${i}`).addClass("m3")
-//     }
-// };
+//charts=======================================================================================================================================
 
 var monthlyTemps = [
```

Comment:
is this supposed to be there? seems like an oversight since it's all the way at the bottom and is only half of an array